sorted_rating.sh

```diff
@@ -0,0 +1,30 @@
+echo enter directory which contains movie folder
+read dir
+final_rate.txt
+for f in $dir/movies/*
+ do
+    oname=${f##*/}
+   # echo $oname
+   name=$(echo ${oname//./+})
+   name=$(echo ${name//_/+})
+   name=$(echo ${name// /+})
+
+   curl -s  http://www.imdb.com/find\?ref_\=nv_sr_fn\&q\=$name\&s\=all |grep -E -0  '/tt\w+' |cut -c72- >out.txt
+
+   cut -c-9 out.txt >  final.txt
+
+   read val <final.txt
```

Comment:
You should leave exactly one space around operators like `|`, `<`, `>`, etc. So this should be rewritten to: `read val < final.txt`